devito/yask/data.py

```diff
@@ -0,0 +1,311 @@
+import sys
+
+import numpy as np
+
+from devito.logger import yask as log
+from devito.tools import as_tuple
+
+from devito.yask.utils import rawpointer
+
+
+class Data(object):
```

Comment:
Note for reviewers: this was formerly called ``YaskGrid``, and it used to be inside ``wrappers.py``.